src/messages/MFSMap.h

```diff
@@ -48,11 +50,11 @@ class MFSMap : public Message {
   void decode_payload() {
     bufferlist::iterator p = payload.begin();
     ::decode(epoch, p);
-    ::decode(encoded, p);
+    ::decode(fsmap, p);
   }
   void encode_payload(uint64_t features) {
     ::encode(epoch, payload);
-    ::encode(encoded, payload);
+    ::encode(fsmap, payload, features);
```

Comment:
So if we're changing the wire encoding like this it certainly needs to go in Jewel. Don't think that will be a problem as we're still merging master in to Jewel; sound good to you?